docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -17,24 +17,40 @@ seo:
   noindex: false # false (default) or true
 ---
 This is not a demo. If you would like an all-in-one demo, please take a look at our [local bundle](https://www.authelia.com/integration/deployment/docker/#local).
-## Assumptions
-We assume these items have already been completed prior to starting this guide.
-- [Docker](https://docs.docker.com/engine/install/) needs to be configured.
+## Assumptions and Adaptation
+This guide makes a few assumptions. These assumptions may require adaptation in more advanced and complex scenarios. We can not reasonablly have examples for every advanced configuration option that exists. Some of these values can be automatically replaced with documentation variables.
+
+{{< sitevar-preferences >}}
+
+We make the following assumptions:
+- [Docker](https://docs.docker.com/engine/install/) is configured correctly.
+- Single Host
+- You will have to adapt all instances of `{{< sitevar name="host" nojs="authelia" >}}` in the URL if:
+  - you're using a different container name
+  - you deployed the proxy to a different location
+- You will have to adapt all instances of `{{< sitevar name="port" nojs="9091" >}}` in the URL if:
+  - you have adjusted the default port in the configuration
+- You will have to adapt the entire URL if:
+  - Authelia is on a different host to the proxy
+- All services are part of the `{{< sitevar name="domain" nojs="example.com" >}}` domain:
+- This domain and the subdomains will have to be adapted in all examples to match your specific domains unless you're
+  just testing, or you want to use that specific domain
+
 ## File Structure
 The first thing we want to do is set up the file structure. Which should look something like this:
 ```text
 📁 project
  ┣ 📁 authelia
  ┃  ┣ 📁 config
- ┃  ┃ ┣ 📄 configuration.yml
- ┃  ┃ ┗ 📄 users.yml
+ ┃  ┃ ┣ 📄 configuration.yaml
+ ┃  ┃ ┗ 📄 users.yaml
  ┃  ┗ 📁 secrets
- ┣ 📄 compose.yml
+ ┣ 📄 compose.yaml
  ┗ 📁 traefik
     ┣ 📁 config
     ┃ ┣ 📄 acme.json
-    ┃ ┣ 📄 static.json
-    ┃ ┗ 📄 traefik.json
+    ┃ ┣ 📄 static.yaml
```

Comment:
```suggestion
    ┃ ┣ 📄 dynamic.yaml
```

This is to align to the other suggestions, it may change if you go with the alternative approach suggested later too.